de_templates/ingest/Ingest_SourceType_SourceName/data_factory/pipelines/ARM_IngestTemplate.json.jinja

```diff
@@ -199,7 +205,48 @@
                                             }
                                         }
                                     }
-                                }
+                                },
+                                {
+                        "name": "correlation_id",
```

Comment:
I think this correlation_id is quite confusing for someone not very technical looking at the pipelines. Maybe pipeline_id would be more suitable? Feel free to resolve though.

Reply:
Changed to `run_id` throughout